examples/app-with-rbac/package.json

```diff
@@ -18,7 +19,10 @@
   "devDependencies": {
     "@babel/core": "^7.21.4",
     "@grafana/eslint-config": "^7.0.0",
+    "@grafana/plugin-e2e": "1.3.1",
+    "@grafana/plugin-meta-extractor": "^0.0.2",
```

Comment:
How did this package end up in here? 🤔 Please remove it as it should not be used anywhere and will produce `generated` properties on plugin.json files which during a plugin submission will be flagged.